src/api/app/services/notification_creator.rb

```diff
@@ -5,23 +5,36 @@ class NotificationCreator
                       'Event::CommentForProject',
                       'Event::CommentForPackage',
                       'Event::CommentForRequest'].freeze
+  CHANNELS = [:web, :rss].freeze
 
   def initialize(event)
     @event = event
   end
 
   def call
     return unless @event.eventtype.in?(EVENTS_TO_NOTIFY)
-    @event.subscriptions.each { |subscription| create_notification_per_subscription(subscription) }
+
+    CHANNELS.each do |channel|
+      @event.subscriptions(channel).each do |subscription|
+        create_notification_per_subscription(subscription, channel)
+      end
+    end
   rescue StandardError => e
     Airbrake.notify(e, event_id: @event.id)
   end
 
   private
 
-  def create_notification_per_subscription(subscription)
-    return if subscription.subscriber && subscription.subscriber.away?
+  def create_notification_per_subscription(subscription, channel)
+    return unless create_notification?(subscription.subscriber, channel)
     params = subscription.parameters_for_notification.merge!(@event.parameters_for_notification)
-    Notification::RssFeedItem.find_or_create_by!(params) # avoid duplication
+    notification = Notification::RssFeedItem.find_or_create_by!(params) # avoid duplication
+    notification.update_attributes("#{channel}": true)
```

Comment:
https://github.com/openSUSE/open-build-service/issues/9441 <- `update_attributes` is getting deprecated. Could you use `update` here?

Reply:
Done